internal/cmd/clone/clone.go

```diff
@@ -3,12 +3,36 @@
 package clone
 
 import (
+	"strings"
+
+	"github.com/gittuf/gittuf/internal/cmd/common"
 	"github.com/gittuf/gittuf/internal/repository"
+	"github.com/gittuf/gittuf/internal/tuf"
 	"github.com/spf13/cobra"
 )
 
 type options struct {
-	branch string
+	branch           string
+	expectedRootKeys PathList
+}
+
+// PathList is a custom type to represent a list of paths
+type PathList []string
+
+// String implements part of the pflag.Value interface.
+func (p *PathList) String() string {
+	return strings.Join(*p, ",")
+}
+
+// Set implements part of the pflag.Value interface.
+func (p *PathList) Set(value string) error {
+	*p = append(*p, value)
```

Comment:
ah, i suppose we need the ** for this append to hold?

Reply:
Yes